RELEASING.md

```diff
@@ -5,23 +5,23 @@ There are two artifacts that make up a Stackdriver Sandbox release:
 - A set of manifests and code, saved as a git tag in this repository
 
 Contributors can use `./make-release.sh`, along with
-[GitHub Actions automation](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/.github/workflows), 
+[GitHub Actions automation](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/main/.github/workflows), 
 to produce both at once.
 
 ## Release Process
 1. run `export NEW_VERSION=vX.Y.Z`
    - see [Version Names](#version-names)
 1. run `./make-release.sh`
    - tip: try running with `--dry-run` or `--no-push` first to do sanity checks
-1. The script will open a new release branch on the origin repository. Create a pull request for the release
+1. The script will open a new release branch on the origin repository. Create a pull request to main for the release
 1. The script will push a git tag to the repo, which should kick off a
-   [`push-tags` CI job](https://github.com/GoogleCloudPlatform/stackdriver-sandbox/blob/master/.github/workflows/push-tags.yml).
+   [`push-tags` CI job](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/main/.github/workflows/push-tags.yml).
    Check that the job completed successfully, and the tagged images appear in the
    [`stackdriver-sandbox-230822` GCR repo](http://console.cloud.google.com/gcr/images/stackdriver-sandbox-230822)
-1. When the PR has been reviewed and thoroughly tested, merge it into master
-   - Don't squash; we must keep the tagged commit in the git history
-1. The merge to master should kick off a
-   [`github-pages` job](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/deployments/activity_log?environment=github-pages)
+1. When the PR has been reviewed and thoroughly tested, merge it into main
+   - Create a merge commit so the history is preserved in main
```

Comment:
command example please

Reply:
This is actually through the GitHub UI. I'll change the wording to make it more clear